rules.md

```diff
@@ -104,9 +81,9 @@ The authoritative source for the Rules are contained in a single document hosted
 
 ## 1.7 Legal
 
-The Incubator is an open-source application and not a legal entity and as such doesn’t own any assets, including any rights to any output, nor does the Incubator apply any restrictions on contributor's usage of their own Task output, apart from the requirement that all output must be open-sourced under MIT license.
+The Incubator is not a legal entity and as such doesn’t own any assets, including any rights to any output, nor does the Incubator apply any restrictions on contributor's usage of their own Task output, apart from the requirement that all output must be open-sourced under MIT license.
 
-The only funding the Incubator App receives is in the form of Dash tokens issued directly from the Dash blockchain, which it redistributes to users who provide verifiable open-source output for predefined tasks approved by Admins, who agree to operate in accordance with the Rules defined in this document.
+The only funding the Incubator receives is in the form of Dash tokens issued directly from the Dash blockchain, which it redistributes to users who provide verifiable open-source output for predefined tasks approved by Admins, who agree to operate in accordance with the Rules defined in this document.
```

Comment:
Traditionally we haven't called Dash a token. I think simply using "Dash" would be sufficient.

Reply:
Good suggestion.  Will update.

Reply:
Fixed in https://github.com/dashincubator/dash-incubator-rules/pull/11/commits/289a9577324c2a6c85bb4101145ffea64c8d231c

Reply:
Fixed typo: https://github.com/dashincubator/dash-incubator-rules/pull/11/commits/410ab24f42233b3a83dc454a162c2a4a50b6e793